rfcs/text/0000-matrix-service.md

```diff
@@ -0,0 +1,418 @@
+
+Ex. GET /filters
+
+```
+[
+    "cell_suspension.provenance.document_id",
+    "genes_detected",
+    "total_umis",
+    "barcode",
+    "emptydrops_is_cell",
+    "specimen_from_organism.provenance.document_id",
+    "cell_suspension.genus_species.ontology",
+    "cell_suspension.genus_species.ontology_label",
+    "donor_organism.provenance.document_id",
+    "donor_organism.human_specific.ethnicity.ontology",
+    "donor_organism.human_specific.ethnicity.ontology_label",
+    "donor_organism.diseases.ontology",
+    "donor_organism.diseases.ontology_label",
+    "donor_organism.development_stage.ontology",
+    "donor_organism.development_stage.ontology_label",
+    "donor_organism.sex",
+    "donor_organism.is_living",
+    "derived_organ_ontology",
+    "derived_organ_label",
+    "derived_organ_parts_ontology",
+    "derived_organ_parts_label",
+    "specimen_from_organism.organ.ontology",
+    "specimen_from_organism.organ.ontology_label",
+    "specimen_from_organism.organ_parts.ontology",
+    "specimen_from_organism.organ_parts.ontology_label",
+    "library_preparation_protocol.provenance.document_id",
+    "library_preparation_protocol.input_nucleic_acid_molecule.ontology",
+    "library_preparation_protocol.input_nucleic_acid_molecule.ontology_label",
+    "library_preparation_protocol.library_construction_method.ontology",
+    "library_preparation_protocol.library_construction_method.ontology_label",
+    "library_preparation_protocol.end_bias",
+    "library_preparation_protocol.strand",
+    "project.provenance.document_id",
+    "project.project_core.project_short_name",
+    "project.project_core.project_title",
+    "analysis_protocol.provenance.document_id",
+    "dss_bundle_fqid",
+    "bundle_uuid",
+    "bundle_version",
+    "file_uuid",
+    "file_version",
+    "analysis_protocol.protocol_core.protocol_id",
+    "analysis_working_group_approval_status"
+]
+```
+
+Ex. Categorical filter: GET /filters/project.project_core.project_short_name
+
+```
+{
+    "cell_counts": {
+        "1M Immune Cells": 782859,
+        "Fetal/Maternal Interface": 546183,
+        "HPSI human cerebral organoids": 156936,
+        "Human Hematopoietic Profiling": 41331,
+        "HumanColonicMesenchymeIBD": 61604,
+        "HumanTissueTcellActivation": 267360,
+        "KidneySingleCellAtlas": 341079,
+        "Reprogrammed_Dendritic_Cells": 15744,
+        "Single cell RNAseq characterization of cell types produced over time in an in vitro model of human inhibitory interneuron differentiation.": 1733,
+        "Single cell transcriptome analysis of human pancreas": 2544,
+        "SingleCellLiverLandscape": 299486,
+        "Tissue stability": 73515,
+        "WongAdultRetina": 41350
+    },
+    "field_description": "A short name for the project.",
```

Comment:
Are field descriptions where possible taken directly from the metadata schema?  Are there any field descriptions for fields defined in the schema which don't match the schema?

Reply:
I believe (1) this is true, and (2) no. @mckinsel can you confirm?